lib/matplotlib/figure.py

```diff
@@ -2298,7 +2298,8 @@ def get_tightbbox(self, renderer, bbox_extra_artists=None):
 
         for ax in self.axes:
             if ax.get_visible():
-                bb.append(ax.get_tightbbox(renderer, bbox_extra_artists))
+                bb.append(ax.get_tightbbox(renderer,
+                         bbox_extra_artists=bbox_extra_artists))
```

Comment:
The alignment of this keyword argument makes me think it is for the append, not the get_tightbbox(), but even that looks wonky. Which PEP8 alignment rule is this?

Reply:
I just did two-tab (8-space) indentation - I don't think it'll fit on one line, though I can check

Reply:
It will not, nor will it work with aligning to the correct opening parenthesis, but you can do this:
```python
                bb.append(                                                      
                    ax.get_tightbbox(renderer,                                  
                                     bbox_extra_artists=bbox_extra_artists))    
```

Reply:
Ummm, OK, but *really* the 8-space indent is allowed by PEP8, and in this case I think looks best:

https://www.python.org/dev/peps/pep-0008/?#indentation

Reply:
I'm meant "will not fit", not anything about whether it's in PEP8.

Reply:
Sorry, @QuLogic, the PEP8 ref was to @WeatherGod above as to why I chose to indent that way (and obviously because aligning with the bracket won't work). 

Your way has a 4-space indent which is a bit funky.  But maybe it's OK?

Reply:
Can also do `bb += [blah]` which takes up less room.  But I kind of hate changing syntax just to make PEP8 happy...

Reply:
That PEP8 link uses 4-space indent for continuation lines; it's only if there's ambiguity that double indent is used (but keep in mind there are no nested function calls in those examples.)

Reply:
I think a list comprehension would be more readable anyway.

Reply:
OK, list comprehension is bearable.

Reply:
Uh, I think (hope?) @timhoffm meant something like
```
        bbox.extend(
            ax.get_tightbbox(renderer, bbox_extra_artists=bbox_extra_artists)
            for ax in self.axes if ax.get_visible())
```

Reply:
Fine